packages/contracts/src/dollar/UbiquityFormulas.sol

```diff
@@ -1,119 +1,123 @@
         public
         pure
-        returns (uint256 _shares)
+        returns (uint256 shares)
     {
         bytes16 unit = uint256(1 ether).fromUInt();
-        bytes16 d = _weeks.fromUInt();
+        bytes16 d = weeks_.fromUInt();
         bytes16 d32 = (d.mul(d).mul(d)).sqrt();
-        bytes16 m = _multiplier.fromUInt().div(unit); // 0.0001
-        bytes16 a = _uLP.fromUInt();
+        ///bytes16 m = multiplier.fromUInt().div(unit); // 0.0001
+        bytes16 a = uLP.fromUInt();
 
-        _shares = m.mul(d32).mul(a).add(a).toUInt();
+        shares = multiplier.fromUInt().mul(d32).mul(a).div(unit).add(a).toUInt();
     }
 
     /// @dev formula bonding
-    /// @param _shares , amount of shares
-    /// @param _currentShareValue , current share value
-    /// @param _targetPrice , target Ubiquity Dollar price
-    /// @return _uBOND , amount of bonding shares
-    /// @notice UBOND = _shares / _currentShareValue * _targetPrice
-    // newShares = A / V * T
-    function bonding(
-        uint256 _shares,
-        uint256 _currentShareValue,
-        uint256 _targetPrice
-    ) public pure returns (uint256 _uBOND) {
-        bytes16 a = _shares.fromUInt();
-        bytes16 v = _currentShareValue.fromUInt();
-        bytes16 t = _targetPrice.fromUInt();
+    /// @param shares , amount of shares
+    /// @param currentShareValue , current share value
+    /// @param targetPrice , target Ubiquity Dollar price
+    /// @return uBOND , amount of bonding shares
+    /// @notice UBOND = shares * targetPrice / currentShareValue
+    // newShares = A * T / V
+    function staking(
+        uint256 shares,
+        uint256 currentShareValue,
+        uint256 targetPrice
+    ) public pure returns (uint256 uBOND) {
+        bytes16 a = shares.fromUInt();
+        bytes16 v = currentShareValue.fromUInt();
+        bytes16 t = targetPrice.fromUInt();
 
-        _uBOND = a.div(v).mul(t).toUInt();
+        uBOND = a.mul(t).div(v).toUInt();
     }
 
     /// @dev formula redeem bonds
-    /// @param _uBOND , amount of bonding shares
-    /// @param _currentShareValue , current share value
-    /// @param _targetPrice , target uAD price
-    /// @return _uLP , amount of LP tokens
-    /// @notice _uLP = _uBOND * _currentShareValue / _targetPrice
-    // _uLP = A * V / T
-    function redeemBonds(
-        uint256 _uBOND,
-        uint256 _currentShareValue,
-        uint256 _targetPrice
-    ) public pure returns (uint256 _uLP) {
-        bytes16 a = _uBOND.fromUInt();
-        bytes16 v = _currentShareValue.fromUInt();
-        bytes16 t = _targetPrice.fromUInt();
+    /// @param uBOND , amount of bonding shares
+    /// @param currentShareValue , current share value
+    /// @param targetPrice , target uAD price
+    /// @return uLP , amount of LP tokens
+    /// @notice uLP = uBOND * currentShareValue / targetPrice
+    // uLP = A * V / T
+    function redeemShares(
+        uint256 uBOND,
```

Comment:
This doesn't seem right to refer to it as `uBOND` for several reasons.
1. The original "uBOND" technically was a non-expiring Ubiquity Credit NFT
2. This is referring to "bonding shares" which is sort of a hidden token for internal accounting. 
3. We renamed "bonding" to "staking"

This should be named with its full name, like stakingShare or something.